components/collapsible-panel/collapsible-panel.js

```diff
@@ -331,6 +335,13 @@ class CollapsiblePanel extends RtlMixin(LitElement) {
 		element.classList.add('has-summary');
 	}
 
+	_normalizeHeadingNumber(number) {
```

Comment:
Nit: could just be a local function defined at the top of this file and not a member function.